src/components/CorporateCampaign/LoanSearch/LoanSearchFilters.vue

```diff
@@ -19,196 +19,220 @@
+							class="loan-filter-controls__filter-type"
+							:all-countries="allCountries"
+							:initial-countries="initialCountries"
+							:selected-countries="selectedCountries"
+							@updated-filters="handleUpdatedFilters"
+						/>
+					</kv-accordion-item>
 
-				<kv-accordion-item
-					class="loan-filters__lightbox-accordian"
-					id="tags-accordian"
-				>
-					<template #header>
-						<h3 class="tw-py-1">
-							Tags
-						</h3>
-					</template>
-					<tag-filter
-						class="loan-filter-controls__filter-type"
-						:all-tags="allTags"
-						:initial-tags="initialTags"
-						:selected-tags="selectedTags"
-						@updated-filters="handleUpdatedFilters"
-					/>
-				</kv-accordion-item>
-			</div>
+					<kv-accordion-item
+						class="loan-filters__lightbox-accordian"
+						id="sectors-accordian"
+					>
+						<template #header>
+							<h3 class="tw-py-1">
+								Sectors
+							</h3>
+						</template>
+						<sector-filter
+							class="loan-filter-controls__filter-type"
+							:all-sectors="allSectors"
+							:initial-sectors="initialSectors"
+							:selected-sectors="selectedSectors"
+							@updated-filters="handleUpdatedFilters"
+						/>
+					</kv-accordion-item>
 
-			<template #controls>
-				<kv-button
-					class="button smallest"
-					@click.native.prevent="applyFilters"
-				>
-					Apply Filters
-				</kv-button>
-			</template>
-		</kv-lightbox>
+					<kv-accordion-item
+						class="loan-filters__lightbox-accordian"
+						id="attributes-accordian"
+					>
+						<template #header>
+							<h3 class="tw-py-1">
+								Attributes
+							</h3>
+						</template>
+						<attribute-filter
+							class="loan-filter-controls__filter-type"
+							:all-attributes="allAttributes"
+							:initial-attributes="initialAttributes"
+							:selected-attributes="selectedAttributes"
+							@updated-filters="handleUpdatedFilters"
+						/>
+					</kv-accordion-item>
+
+					<kv-accordion-item
+						class="loan-filters__lightbox-accordian"
+						id="tags-accordian"
+					>
+						<template #header>
+							<h3 class="tw-py-1">
+								Tags
+							</h3>
+						</template>
+						<tag-filter
+							class="loan-filter-controls__filter-type"
+							:all-tags="allTags"
+							:initial-tags="initialTags"
+							:selected-tags="selectedTags"
+							@updated-filters="handleUpdatedFilters"
+						/>
+					</kv-accordion-item>
+				</div>
+
+				<template #controls>
+					<kv-button
+						variant="primary"
+						@click.native.prevent="applyFilters"
+					>
+						Apply Filters
+					</kv-button>
+				</template>
+			</kv-lightbox>
+		</div>
 	</div>
 </template>
 
 <script>
 import _isEqual from 'lodash/isEqual';
 import _sortBy from 'lodash/sortBy';
 import gql from 'graphql-tag';
+import { mdiGridLarge, mdiLandRowsHorizontal } from '@mdi/js';
 import KvButton from '@/components/Kv/KvButton';
 import KvChip from '@/components/Kv/KvChip';
 import KvLightbox from '@/components/Kv/KvLightbox';
```

Comment:
This import should be converted to use kv-components along with the signature in the template.